test/views/components/LateSubmissionPenaltySummaryCardSpec.scala

```diff
@@ -740,12 +740,12 @@ class LateSubmissionPenaltySummaryCardSpec extends SpecBase with ViewBehaviours
 
       "set the correct hidden span for a lurking point with no return submitted" in {
         doc.select("a").get(0).ownText() shouldBe "Check if you can appeal"
-        doc.select("span").text() shouldBe "for penalty on late VAT return due 30 October 2069"
+        doc.select("span").get(1).text() shouldBe "for penalty on late VAT return due 30 October 2069"
```

Comment:
This check is a vulnerable to fail or prove the wrong thing:
```suggestion
        doc.select("a span").get(0).text() shouldBe "for penalty on late VAT return due 30 October 2069"
```